src/serializers/deployment-request.js

```diff
@@ -1,14 +1,6 @@
 const JSONAPISerializer = require('jsonapi-serializer').Serializer;
 
 module.exports = new JSONAPISerializer('deployment-requests', {
-  attributes: ['from', 'to'],
-  from: {
-    ref: 'id',
-    included: false,
-  },
-  to: {
-    ref: 'id',
-    included: false,
-  },
-  typeForAttribute: () => 'environments',
+  attributes: ['type', 'from', 'to'],
+  typeForAttribute: () => 'deployment-requests',
```

Comment:
This is useless. There is no attribute with not foreignKey so typeForAttribute will never be called. It can be removed